pyface/ui/wx/grid/grid.py

```diff
@@ -226,39 +226,39 @@ def __init__(self, parent, **traits):
         grid.Bind(grid_movers.EVT_GRID_COL_MOVE, self._on_col_move)
         grid.Bind(grid_movers.EVT_GRID_ROW_MOVE, self._on_row_move)
 
-        smotc = self.model.on_trait_change
-        otc = self.on_trait_change
-        smotc(self._on_model_content_changed, "content_changed")
-        smotc(self._on_model_structure_changed, "structure_changed")
-        smotc(self._on_row_sort, "row_sorted")
-        smotc(self._on_column_sort, "column_sorted")
-        otc(self._on_new_model, "model")
+        smobs = self.model.observe
+        obs = self.observe
```

Comment:
Please let's not do this - it is not at all obvious what `smobs` and `obs` are - and how they are different. The temporary variables are just not worth the long lines/wrapped lines that we will need to deal with.